cyberdrop_dl/scraper/scrape_mapper.py

```diff
@@ -51,6 +51,8 @@ def group_count(self) -> int:
     def start_scrapers(self) -> None:
         """Starts all scrapers."""
         self.existing_crawlers = get_crawlers(self.manager)
+        if not self.manager.config_manager.global_settings_data.general.enable_generic_crawler:
+            _ = self.existing_crawlers.pop(".")
```

Comment:
```suggestion
            self.existing_crawlers.pop(".")
```
Return value is unused

Reply:
`_` is to indicate "I know this returns something, but I don't need it,". This makes it clear that we are ignoring the return value on purpose, not by accident.